### PR TITLE
feat(gallery): Add BubbleXY sample

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

File: demos/gallery/samples/chart/BubbleXY.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let categories = {};
+let falsey_defaults = {};
+new BubbleXY()._publishedProperties.forEach(prop => {
+    if (!categories[prop.type]) {
+        categories[prop.type] = 0;
+        falsey_defaults[prop.type] = 0;
+    }
+    categories[prop.type]++;
+    if(!prop.defaultValue)falsey_defaults[prop.type]++;
+});
```
Comment (GordonSmith) [inline demos/gallery/samples/chart/BubbleXY.js]: This code confuses me - so I suspect it would confuse a user, not a great example...

Comment (jbrundage) [reply demos/gallery/samples/chart/BubbleXY.js]: I was trying to add an example that uses information from the widget as its own data source. I should probably just add some standard examples.

Comment (GordonSmith) [reply demos/gallery/samples/chart/BubbleXY.js]: Yup - lets make everything very easy to understand...

**Comment by jbrundage** [comment]
@GordonSmith this has been revised

**Pull request merged**